### ISSUE TITLE
Implementation details of CUB200

### ISSUE THREAD

**Issue by RHe502** [dipamgoswami/ADC#3]
Hi!
Thank you for your impressive work! I am now reproducing the results of ADC on CUB200 but I cannot get similar results to those reported in the paper. For example, for T=10, I can only obtain the result of last accuracy of 41.92, which is much lower than the reported results of 51.78 in the paper. 

I follow the details of implementation in supplimentary materials and the experiment setting is list below. 

init_epoch = 200
init_lr = 0.01
init_milestones = [60, 120, 160]
init_lr_decay = 0.1
init_weight_decay = 0.0005

epochs = 100
lrate = 0.005
milestones = [45, 90]
lrate_decay = 0.1
batch_size = 128
weight_decay = 2e-4
num_workers = 8
T = 2
lamda = 20

Is there any problem for this setting? Did I use a wrong set of parameters? Could you please provide the detailed paramerters?

Thanks a lot!

**Comment by dipamgoswami** [comment]
Hi,
The training settings for CUB200 (T=10) looks the same as stated in the paper.
You should get a last accuracy of 41.96 if you only do LwF. If you use the ADC method of correcting prototypes, the last accuracy should be 57.97.  

Can you provide more details of your issue? Can you check if you are generating adversarial samples and using them or not? It looks like you are getting the LwF accuracy.

**Comment by RHe502** [comment]
Thank you for your response! I check the setting of expeiments and I correctly use the ADC. For the LwF, with the same seed, the result of final accuracy will be 34.1. It seems that the ADC has already improved over the LwF. Does this issue come from the environment or GPU? I did these experiments on a single NVIDIA 3090 GPU with torch 1.12.0. Since the dependencies provided in this repository seems have a minor mistake, I use the torch 1.12.0. For example, "torch 1.81" seems a typo and torchvision 0.6.0 can be only supported by torch 1.5.0. Could you please provide more information of the dependencies then I can further check my implementation. Thank you!

**Comment by dipamgoswami** [comment]
Hi, thanks for the details. I will try to run the setting of CUB200 again and see if I can replicate the results. Give me a few days time, I am a bit occupied immediately. I will share the details of dependencies too as soon as possible.
